src/main/java/com/mosaicchurchaustin/oms/services/OrderService.java

```diff
@@ -281,4 +286,42 @@ private void addItemsToOrder(final OrderEntity orderEntity, List<OrderItemReques
         }
     }
 
+    public String tryGetSerializedOrder(OrderEntity order) {
+        try {
+            return getSerializedOrder(order);
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    private String getSerializedOrder(OrderEntity order) {
```

Comment:
I think I would move the Order serialization stuff to the GroupMeService, since it's (at least currently) specifically for that integration.